Fix docs: in labelname, convert to underscore (#12555)
prometheus/prometheus: docs/configuration/configuration.md

* `<float>`: a floating-point number
* `<host>`: a valid string consisting of a hostname or IP followed by an optional port number
* `<int>`: an integer value

> * `<labelname>`: a string matching the regular expression `[a-zA-Z_][a-zA-Z0-9_]*`. Any other special character in the source label is converted to an underscore. For example, the label `app.kubernetes.io/name` should be written as `app_kubernetes_io_name`.

Reviewer: ```suggestion
* `<labelname>`: a string matching the regular expression `[a-zA-Z_][a-zA-Z0-9_]*`. Any other special character in the source label should be converted to an underscore. For example, the label `app.kubernetes.io/name` should be written as `app_kubernetes_io_name`.
```

* `__meta_docker_container_id`: the id of the container
* `__meta_docker_container_name`: the name of the container
* `__meta_docker_container_network_mode`: the network mode of the container

> * `__meta_docker_container_label_<labelname>`: each label of the container, but you need to convert any dot, dash, and slash to an underscore

Reviewer: ```suggestion
* `__meta_docker_container_label_<labelname>`: each label of the container, with any unsupported characters converted to an underscore
```
